README.md

```diff
@@ -136,17 +137,16 @@ If you have installed Chocolatey, you can complete these steps to set up JDK, Ap
 > On OS X systems, you can develop, build, and deploy NativeScript projects that target iOS and Android.
 
 * OS X Mavericks
-* [Node.js 0.10.26] or a later stable official release
+* [Node.js 0.10.26][Node.js 0.10.26] or a later stable official release
 * For iOS development
-	* [iOS 7.0 SDK][12] or a later stable official release
-	* [Xcode 5][12] or a later stable official release
+	* [Xcode 6][12] or a later stable official release
 	* [Xcode command-line tools][12]
```

Comment:
If I remember correctly, only 64 bit Mono works. Someone from the NS team should confirm.